Library/Formula/jenkins.rb

```diff
@@ -58,12 +64,12 @@ def caveats; <<-EOS.undent
   test do
     ENV["JENKINS_HOME"] = testpath
     pid = fork do
-      exec "#{bin}/jenkins"
+      exec "java -Djava.io.tmpdir=#{testpath} -jar #{opt_libexec}/jenkins.war"
```

Comment:
@dunn I now see that I can pass `java.io.tmpdir` as an argument to `bin.write_jar_script()`, instead of this change. That would also apply to the installed app, not just the test. I'll do it that way instead. Cool?